sdv/single_table/base.py

```diff
@@ -559,7 +559,7 @@ def _filter_conditions(sampled, conditions, float_rtol):
             if column_values.dtype.kind == 'f':
                 distance = value * float_rtol
                 sampled = sampled[np.abs(column_values - value) <= distance]
-                sampled[column] = value
+                sampled.loc[:, column] = value
```

Comment:
I thought you said this didn't work?

Reply:
I think it was a local issue that was caching the old code for some reason, after I reinstalled everything I did not see the error anymore...

Reply:
Don't mind.. in collab still fails 😮‍💨

Reply:
Seems to be a caching issue. I was able to run it in colab without the error